Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -2133,6 +2180,9 @@ String HTMLInputElement::convert_number_to_string(double input) const
     if (type_state() == TypeAttributeState::Month)
         return convert_month_number_to_string(input);
 
+    if (type_state() == TypeAttributeState::Week)
+        return convert_week_number_to_string(input);
```

Comment:
These conversion methods have confusing names; I'd expect `convert_number_to_week_string` and `convert_number_to_month_string` instead.

Reply:
indeed, much better